### ISSUE TITLE
Password for Web Interface

### ISSUE THREAD

**Issue by mafrmt00** [GeekMagicClock/smalltvpro#11]
I just bought a second device to use at my workplace. But the I noticed that I can not protect access to the device.
If it is used in a public wifi, everyone could access the device.
Please add the possibiliy to protect the web interface with a password.

**Comment by GeekMagicClock** [comment]
Hi, reasonable needs, but maybe most people will use this device at home or company.

**Comment by mafrmt00** [comment]
Yes, at work! I don't want my coworkers to mess with the displayed pictures. The would replace my family pictures with nasty stuff!

**Comment by GeekMagicClock** [comment]
OK, with take that into consideration in future versions!

**Comment by seocrow** [comment]
The password being in cleartext could also be an issue. Better to hide behind **** if possible

**Comment by GeekMagicClock** [comment]
Hi, password is hidden now.

**Comment by mafrmt00** [comment]
> Hi, password is hidden now.

The password for the WLAN is hidden now.
This has nothing to do with the ticket.

A login password should of course be hidden while typing and stored only as hash (salted) on the device.

**Comment by GeekMagicClock** [comment]
> > Hi, password is hidden now.
> 
> The password for the WLAN is hidden now. This has nothing to do with the ticket.
> 
> A login password should of course be hidden while typing and stored only as hash (salted) on the device.

Yes, you are right, there has to be authentication to avoid this, adding authentication is easy, but the admin's password also could be forgot, so there has to be a reset button/method, but I have no idea for reset yet.

**Comment by mafrmt00** [comment]
> ...so there has to be a reset button/method, but I have no idea for reset yet.

There is one button (Touch). What about pressing it while powering on the device and keep it pressed for 10sec to do a factory reset?
